ipa-core/src/cli/crypto/mod.rs

```diff
@@ -0,0 +1,198 @@
+mod decrypt;
+mod encrypt;
+
+pub use decrypt::DecryptArgs;
+pub use encrypt::EncryptArgs;
+
+#[cfg(test)]
+mod sample_data {
+    use std::{io, io::Write, sync::OnceLock};
+
+    use hpke::{Deserializable, Serializable};
+    use rand::thread_rng;
+    use tempfile::NamedTempFile;
+
+    use crate::{
+        cli::CsvSerializer,
+        hpke::{IpaPrivateKey, IpaPublicKey},
+        test_fixture::{ipa::TestRawDataRecord, EventGenerator, EventGeneratorConfig},
+    };
+
+    /// Keys that are used in crypto tests
+    #[derive(Clone)]
+    pub(super) struct TestKeys {
+        inner: [(IpaPublicKey, IpaPrivateKey); 3],
```

Comment:
wouldn't `keys` be more descriptive than `inner`? maybe `TestKeys.keys` is too repetitive...

Reply:
I am probably abusing `inner` too much, I agree that `keys` is better if we add anything else inside after. I'll stick with `key_pairs`